main.py

```diff
@@ -0,0 +1,111 @@
+import datetime
+import itertools
+
+from InstagramAPI import InstagramAPI
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from models import Base, InstagramUsers
+
+user_name = 'case_iphone_lviv2019'
+password = 'игвяеуіе123'
+
+
+class GetInstagramInfo:
+    def __init__(self, username, password):
+        self.api = InstagramAPI(username=username, password=password)
+        self.api.login()
+
+    def get_user_id(self, name):
+        """Returns id user."""
+        self.api.searchUsername(name)
+        return self.api.LastJson["user"]["pk"]
+
+    def get_followers(self, user_id):
+        """Returns list of user's followers."""
+        followers = self.api.getTotalFollowers(user_id)
+        return [follower['username'] for follower in followers]
+
+    def get_followings(self, user_id):
+        """Returns list of user's followings."""
+        followings = self.api.getTotalFollowings(user_id)
+        return [following['username'] for following in followings]
+
+    def create_database(self):
+        engine = create_engine('sqlite:///users.db')
+        Base.metadata.create_all(engine)
+        Session = sessionmaker(bind=engine)
+        return Session()
+
+    def time_now(self):
+        return datetime.datetime.now().strftime('%Y-%m-%d %H:%M')
+
+    def save_db(self, connect_database, followers, followings):
+        """Written followers and followings user to database"""
+        session = connect_database
+        now = self.time_now()
+        for (followings, followers) in itertools.zip_longest(followings, followers, fillvalue=''):
+            text = InstagramUsers(followings=followings, followers=followers, status="new", time=str(now))
+            session.add(text)
+        session.commit()
+
+    def database_followings(self, connect_database):
+        """Getting all followings of the database"""
+        session = connect_database
+        database_followings = session.query(InstagramUsers.followings).all()
+        return [y for x in database_followings for y in x]
+
+    def database_follower(self, connect_database):
+        """Getting all followers of the database"""
+        session = connect_database
```

Comment:
It's easier to create your session in the constructor instead of passing it to each method as a parameter.

Reply:
done